app/docs/enterprise/0.32-x/installation/ubuntu.md

```diff
@@ -0,0 +1,74 @@
+---
+title: How to install Kong Enterprise and PostgreSQL onto Ubuntu
+---
+
+## Install Kong Enterprise
+
+curl the EE binary for Ubuntu (or Debian) from Bintray to your Ubuntu server
```

Comment:
I think we should have a code block after this with an example `curl` statement.